test/fixtures/rules/std_rule_cases/L003.yml

```
@@ -643,3 +643,11 @@ test_pass_tsql_declare_indent:
   configs:
     core:
       dialect: tsql
+
+test_pass_tsql_set_indent:
+  pass_str: |
+    SET @prv_qtr_1st_dt = CAST(@last_qtr_yr AS VARCHAR(4)) + '-' +
+        CAST(@last_qtr_first_mn AS VARCHAR(2)) + '-01'
+  configs:
+    core:
+      dialect: tsql
```

Could we also add an example with commas?

For example (untested):

```suggestion
test_pass_tsql_set_indent:
  pass_str: |
    SET @prv_qtr_1st_dt = CAST(@last_qtr_yr AS VARCHAR(4)) + '-' +
        CAST(@last_qtr_first_mn AS VARCHAR(2)) + '-01'
  configs:
    core:
      dialect: tsql

test_pass_tsql_set_indent_multiple_params:
  pass_str: |
    SET @param1 = 1,
        @param2 = 2
  configs:
    core:
      dialect: tsql
```

---

Oops. Looking TSQL (or at least SQLFluff’s understanding of TSQL) doesn’t support multiple sets! Sorry.

---

@tunetheweb turns out that's not valid TSQL.  SET variable statements are limited to a single variable.